editor/editor_resource_picker.cpp

```diff
@@ -505,7 +505,16 @@ bool EditorResourcePicker::_is_drop_valid(const Dictionary &p_drag_data) const {
 	Ref<Resource> res;
 	if (drag_data.has("type") && String(drag_data["type"]) == "script_list_element") {
 		ScriptEditorBase *se = Object::cast_to<ScriptEditorBase>(drag_data["script_list_element"]);
-		res = se->get_edited_resource();
+		if (se) {
+			res = se->get_edited_resource();
+		} else {
+			EditorHelp *eh = Object::cast_to<EditorHelp>(drag_data["script_list_element"]);
```

Comment:
I don't think we need this `else` block. We can't really accept anything from the `EditorHelp` as a valid drop item in the resource picker. It will just crash again `drop_data_fw` where we cast it to `ScriptEditorBase` again.

Reply:
Ah right, it's true that it should return false in that case, and it's will be fine to fix this issue.

Reply:
Done